pkg/webhooks/trainingruntime_webhook.go

```diff
@@ -60,8 +61,18 @@ func validateReplicatedJobs(rJobs []jobsetv1alpha2.ReplicatedJob) field.ErrorLis
 		Child("replicatedJobs")
 	var allErrs field.ErrorList
 	for idx, rJob := range rJobs {
-		if rJob.Replicas != 1 {
-			allErrs = append(allErrs, field.Invalid(rJobsPath.Index(idx).Child("replicas"), rJob.Replicas, "always must be 1"))
+		if rJob.Template.Labels == nil {
+			continue
+		}
+
+		if labelAncestor, ok := rJob.Template.Labels[constants.LabelTrainJobAncestor]; ok {
+			if labelAncestor != constants.AncestorTrainer && labelAncestor != constants.ModelInitializer && labelAncestor != constants.DatasetInitializer {
+				continue
+			}
+
+			if rJob.Replicas != 1 {
+				allErrs = append(allErrs, field.Invalid(rJobsPath.Index(idx).Child("replicas"), rJob.Replicas, "always must be 1"))
+			}
```

Comment:
```suggestion
		if labelAncestor, ok := rJob.Template.Labels[constants.LabelTrainJobAncestor]; ok && ancestors.Has(labelAncestor) &&  rJob.Replicas != 1 {
			allErrs = append(allErrs, field.Invalid(rJobsPath.Index(idx).Child("replicas"), rJob.Replicas, "always must be 1"))
```

Reply:
Thanks, I have merged the conditions here.